colin-api/src/colin_api/models/business.py

```diff
@@ -46,6 +46,12 @@ class TypeCodes(Enum):
         BC_COMP = 'BC'
         ULC_COMP = 'ULC'
 
+    class CorpFrozenTypes(Enum):
```

Comment:
Could you expand on the description of what the codes are.

Reply:
Yes, good point! Thanks!

Reply:
I've changed it.

Reply:
I don't see anything, maybe a missing push?
when would I choose, or use:
```
        COMPANY_FROZEN = 'C'
        SELF_SERVED_FROZEN = 'S'
```

Reply:
Please see https://github.com/bcgov/lear/pull/1138/files#r611738801

Reply:
I'm looking for something like:

```
 """Render an Enum of the Corporation Frozen Type Codes.

     The following frozen states are used for:
     COMPANY_FROZEN is when the Registrar has frozen a company
     SELF_SERVED_FROZEN is when the company has ????
 """
 ```

Reply:
SELF_SERVED_FROZEN  is a legacy code and is deprecated. I updated the comment.